packages/roosterjs-editor-plugins/lib/plugins/ContentEdit/features/markdownFeatures.ts

```diff
@@ -106,13 +106,12 @@ function handleMarkdownEvent(
                     textContentRange.startOffset + 1
                 );
 
-                // set the removal range to include the typed last character.
-                const lastIndex: number = (range.endContainer as Text).length;
-                range.setEnd(range.endContainer, lastIndex);
+                const text = textContentRange.extractContents().textContent;
+                const textNode = editor.getDocument().createTextNode(text);
```

Comment:
`text` can be null, should we add a null check?

Reply:
this range is calculated `cacheGetRangeForMarkdownOperation`, then if the text is null, the range is null and the event will not be handled.